src/natcap/invest/datastack.py

```diff
@@ -85,10 +85,17 @@ def _collect_spatial_files(filepath, data_dir):
     # If the user provides a mutli-part file, wrap it into a folder and grab
     # that instead of the individual file.
 
+    # get just the name of the file without the extension
+    filename = os.path.basename(os.path.splitext(filepath)[0])
+
     with utils.capture_gdal_logging():
         raster = gdal.OpenEx(filepath, gdal.OF_RASTER)
         if raster is not None:
-            new_path = tempfile.mkdtemp(prefix='raster_', dir=data_dir)
+            # give the folder a descriptive name with a unique
+            # random suffix to avoid name conflicts
+            new_path = tempfile.mkdtemp(
+                prefix='%s_raster_' % filename, 
```

Comment:
This might be nitpicky but I feel like we've slowly been adopting f-strings for formatting. I think it reads a little bit nicer. Not a big deal.

Reply:
I saw some other old-style formatted strings in there, but I agree it's probably better to switch to `.format()`. Just fixed these